test/winston-graylog2-test.js

```diff
@@ -59,26 +59,26 @@ describe('winstone-graylog2', function() {
       assert.ok(typeof winstonGraylog2.log === 'function');
     });
 
-    it('should have prelog function', function() {
+    it('should have formatter function', function() {
       let winstonGraylog2 = new WinstonGraylog2();
-      assert.ok(typeof winstonGraylog2.prelog === 'function');
+      assert.ok(typeof winstonGraylog2.formatter === 'undefined');
```

Comment:
@surbhimahajan-wingify test case description is the following: **'should have formatter function'**. But you check the opposite:

```js
assert.ok(typeof winstonGraylog2.formatter === 'undefined');
```

That means that `winstonGraylog2.formatter` is equal to `undefined`. This test is just misleading. Fix it please.

Reply:
@surbhi-mahajan check this comment too please